omod/src/main/webapp/resources/htmlforms/Covid19-VaccinationTracking.xml

```diff
@@ -315,22 +476,26 @@ formLabel="Covid-19 Vaccination Tracking Form">
                   </div>
                   <div class="card-body">
                     <div class="row">
-                      <div class="col-md-4">
+                      <div id="fully-vaccinated" class="col-md-4">
                           <label>Comment why the client is not/ fully vaccinated</label>
-                          <obs id="not-fully-vaccinated" conceptId="166700" answerConceptIds="166699,166692,166693,166694,166695,166696,166697,166698"
+                          <obs id="not-fully-vaccinated" conceptId="166700" answerConceptIds="166699,166692,166693,166694,166695,166696,166697,166698,90002"
                                answerLabels="Vaccine out of stock, Does not trust the vaccine, Not eligible for the vaccine, Not aware of Covid-19 vaccine, Very long lines, Not Accessible,
-                                Does not know where to get the Covid-19 vaccine, Does not want to get the Covid-19 vaccine" />
+                                Does not know where to get the Covid-19 vaccine, Does not want to get the Covid-19 vaccine,Others Specify" />
                       </div>
-                      <div class="col-md-4">
+                      <div id="referred-for-vaccination" class="col-md-4">
                           <label>Referred for vaccination</label>
                           <obs id="referred-for-vaccination" conceptId="166691" answerConceptIds="90003,90004" answerLabels="Yes, No" style="radio" class="horizontal" />
                       </div>
                         <div id="reason-for-refusing-referral-container" class="col-md-4">
                             <label>Reason client does not want to be referred</label>
-                            <obs id="reason-for-refusing-referral" conceptId="166690"/>
+                            <obs id="reason-for-refusing-referral" conceptId="166690" style="textarea"/>
                         </div>
                     </div>
                     <div class="row">
+                        <div id="other-reason-client-not-fully-vaccinated-container" class="col-md-4">
+                            <label>Other Reason</label>
+                            <obs id="other-reason-client-not-fully-vaccinated" conceptId="166739" style="textarea"/>
```

Comment:
Concept '166739' does not exist, add other to 166700 and create other reason concept to use in this section